roles/lvm/tasks/volume.yml

```diff
@@ -9,6 +9,16 @@
     - docker
     - bootstrap
 
+- name: update lvg ansible module
+  connection: local
+  run_once: yes
+  get_url:
+    url: https://raw.githubusercontent.com/ansible/ansible-modules-extras/devel/system/lvg.py
+    dest: "{{ playbook_dir }}/library/lvg.py"
+  tags:
+    - docker
+    - bootstrap
```

Comment:
On Wed, Dec 02, 2015 at 12:32:28PM -0800, Brian Hicks wrote:

> > +  tags:
> > +    - docker
> > +    - bootstrap
> 
> `bootstrap` tasks are for when machine are being bootstrapped with Packer. Could you remove this, please?
> 

this target should run on both cases, it download important file
which we can't keep in our repo due licensing.

or 'packer' is only for creating vagrant image? (which build w/o
lvm)